packages/terra-clinical-item-comment/docs/themeable-variables.md

```diff
@@ -0,0 +1,8 @@
+## Themeable Variables
+
+Variables to `theme` the Terra Clinical ItemComment component.
+
+| Variable Name | Default Value | Usage |
+|---|---|---|
+| `$terra-clinical-item-comment-color` | `$terra-grey-70` | Sets the font color of comment.|
+| `$terra-clinical-item-comment-icon-color` | `$terra-grey-30` | Sets the color of the icon. |
```

Comment:
Do we want to carry these over? I think we were going to hold off on adding themeable variables until we had a better idea of what actually needs to be themed.